ci-operator/config/openshift/assisted-service/openshift-assisted-service-ocm-2.3.yaml

```diff
@@ -12,8 +12,8 @@ base_images:
     namespace: openshift
     tag: assisted-installer-controller
   assisted-service-scripts:
-    name: assisted
-    namespace: ocp-kni
+    name: ocm-2.5
```

Comment:
``ocm-2.3``, no?

Reply:
This is because we don't generate this on ocm-2.3 branch. This is for the ZTP scripts in deploy/operator/.. need to move these things out of here.